AssetLoader/interface/GLTFBuilder.hpp

```diff
@@ -461,6 +461,7 @@ Mesh* ModelBuilder::LoadMesh(const GltfModelType& GltfModel,
             IndexStart,
             IndexCount,
             VertexCount,
+            VertexStart,
```

Comment:
Since for indexed primitives, start vertex is baked into the index buffer, I suggest setting it to zero to avoid confusion.